meta-lxatac-bsp/recipes-bsp/barebox/files/patches/0103-ARM-stm32mp-add-Linux-Automation-TAC-Generation-3.patch

```diff
@@ -604,17 +604,8 @@ index 8a34d15e9005..1fac9bb60cfb 100644
  &gpu {
  	status = "disabled";
  };
-@@ -148,7 +225,7 @@ adc@0 {
- 		compatible = "ti,lmp92064";
- 		reg = <0>;
- 
--		reset-gpios = <&gpioa 4 GPIO_ACTIVE_HIGH>;
-+		reset-gpios = <&gpioa 4 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
```

Comment:
Why is this being removed?

Reply:
This change has made it into the mainline Linux https://github.com/torvalds/linux/commit/1d4e8b91d242c4afa1970ca6f7e850aab66bf452 and mainline barebox https://github.com/barebox/barebox/commit/a0991ad989b2b678aecf1ba85ee6daad469aff89 .